exersices/ex01/letter_counts.py

```diff
@@ -0,0 +1,18 @@
+import collections as c
+#Imported to use Counter() function. A Counter is a dict subclass for counting hashable objects.
+#It is an unordered collection where elements are stored as dictionary keys and their counts are stored as dictionary values.
+#Counts are allowed to be any integer value including zero or negative counts.
+
+def letter_freq(text):
+    text = text.lower() #Making string lower case
```

Comment:
Although comments are appreciated, they should only be used for things that aren't apparent from reading the code itself